features/deep-linking/deep-link.store.ts

```diff
@@ -0,0 +1,57 @@
+import { create } from "zustand"
```

Comment:
can you add a small comment block here explaining why we need this? Maybe couple of examples of why this is needed